src/dump.js

```diff
@@ -1,263 +1,260 @@
+import { inArray } from "./core";
```

Comment:
Can extract `inArray` from core into a standalone module to avoid the circular dependency here.